src/components/Schedule/Data.jsx

```diff
@@ -3,86 +3,114 @@ export const startTime = 'August 10, 2021 18:00:00';
 // Date strings are formatted as "YYYY-MM-DDTHH:mm:ss.sss+-hh:mm"
 export const firstEvents = [
   {
-    name: '05:00PM - Kickoff via Discord (opening-kickoff)',
-    endTime: '2021-08-11T18:00:00.000-04:00',
+    name: '05:00PM - Kickoff via Discord',
+    endTime: '2021-09-24T18:00:00.000-04:00',
   },
   {
-    name: '06:00PM - START HACKING!',
-    endTime: '2021-08-11T18:01:00.000-04:00',
+    name: '06:00PM - Team Building via Discord',
+    endTime: '2021-09-24T18:30:00.000-04:00',
   },
   {
-    name: '06:00PM - Team Building via Discord (#teammate-search)',
-    endTime: '2021-08-11T19:00:00.000-04:00',
+    name: '06:00PM - START HACKING!!!',
+    endTime: '2021-09-24T19:00:00.000-04:00',
   },
   {
-    name: '07:00PM - SB WICS Presents Intro. To Android Studio',
-    endTime: '2021-08-11T20:00:00.000-04:00',
+    name: '07:00PM - Softheon Event',
+    endTime: '2021-09-24T20:00:00.000-04:00',
   },
+  /*
   {
-    name: '08:00PM - SBCS Presents \'Gateway to Github\'',
-    endTime: '2021-08-11T21:00:00.000-04:00',
+    name: '08:00PM - ',
+    endTime: '2021-09-24T21:00:00.000-04:00',
   },
   {
-    name: '09:00PM - Ph.D. CS Student Panel Discussion',
-    endTime: '2021-08-11T22:00:00.000-04:00',
+    name: '09:00PM - ',
+    endTime: '2021-09-24T22:00:00.000-04:00',
   },
   {
-    name: '10:00PM - MLH presents !Light',
-    endTime: '2021-08-11T23:00:00.000-04:00',
+    name: '10:00PM - ',
+    endTime: '2021-09-24T23:00:00.000-04:00',
   },
+  */
   {
-    name: '11:00PM - Silicon Valley Kahoot',
-    endTime: '2021-08-12T00:00:00.000-04:00',
+    name: '11:00PM - Among Us',
+    endTime: '2021-09-25T00:00:00.000-04:00',
   },
 ];
 
 export const secondEvents = [
```

Comment:
Common practice for constant variables is to use all capitals such as SECOND_EVENTS.